inc/azure_macro_utils/macro_utils.h

```diff
@@ -153,53 +181,23 @@ static const char* MU_C2(enumName,Strings)(enumName value)         \
 // this macro returns the number of enum values (taking into account that an invalid value is generated)
 #define MU_ENUM_VALUE_COUNT(...) (MU_COUNT_ARG(__VA_ARGS__) + 1)
```

Comment:
>+ 1 [](start = 60, length = 3)

isnt there some macro that does "INC" ? This does not expand to a single number, it expands to (4 + 1) for example

Reply:
Sure, changed.

---
In reply to: [328357999](https://github.com/Azure/azure-macro-utils-c/pull/30#discussion_r328357999) [](ancestors = 328357999)